sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -396,8 +396,6 @@ private[joins] class UnsafeHashedRelation(
     val nKeys = readLong()
     val nValues = readLong()
     // This is used in Broadcast, shared by multiple tasks, so we use on-heap memory
-    // TODO(josh): This needs to be revisited before we merge this patch; making this change now
-    // so that tests compile:
```

Comment:
It seems that this was a reminder for the author to revisit this part of the code before merging, but it was forgotten to be deleted when the code was merged?